codejail/tests/test_jail_code.py

```diff
@@ -582,3 +581,32 @@ def test_crash_proxy(self):
             pid = proxy.PROXY_PROCESS.pid
             self.assertNotIn(pid, pids)
             pids.add(pid)
+
+
+class TestPython3JailCode(helpers.Python3Mixin, unittest.TestCase):
+    """
+    Test that python 3 codejails can run jail_code
+    """
+
+    def test_jail_code(self):
+        result = self.python3_jail.jail_code('print("Huzzah")')
+        self.assertEqual(result, JailResult(status=0, stdout='Huzzah\n', stderr=''))
+
+    def test_jail_code_error(self):
+        result = self.python3_jail.jail_code('print "Huzzah"')
+        stderr = textwrap.dedent("""\
+              File "jailed_code", line 1
+                print "Huzzah"
+                             ^
+            SyntaxError: {}
+        """)
+
+        try:
+            self.assertEqual(result, JailResult(status=1, stdout='', stderr=stderr.format('invalid syntax')))
+        except AssertionError:
```

Comment:
This looks confusing.  I would deal with version differences in the error message by making smaller assertions, and either only checking for the part up to "SyntaxError:", or using a regex to accommodate the variation.